### ISSUE TITLE
[Bug]: Chromaprint-based intro detection broken with 1.10.10.11

### ISSUE THREAD

**Issue by DrMcCoy** [intro-skipper/intro-skipper#406]
### Self service debugging

- [X] I use Jellyfin 10.10.3 (or newer) and my permissions are correct

### Describe the bug

Since 1.10.10.11 (maybe also 1.10.10.10?), intro fingerprint-based intro detection took a turn for the worse, setting it very often to just from 0 to 0. Previously, it worked fine.

I noticed it because a newly added anime episode wouldn't show a skip button. Looking at the metadata of that episode, the intro wasn't detected, when it was fine for the previous epsiodes. I then proceeded to erase the fingerprint for the whole season (including cache), and reran intro detection. When it finished, only the first three episodes had intro segments, when previously it was all but the newest one (8 in total). That at least limits the bug to be at most a week old, because last week's episode was fine previously.

The files in question don't have any chapters set.

I also noticed that the detection was pretty quick. 27 seconds for all 9 episodes. It *feels* like it used to be slower, but I may very well be wrong there (the system is a Synology NAS, with a CPU that includes a GPU). There's no errors in the Jellyfin log. (Dunno if there's other logs that Intro Skipper writes.)

It's not only show either. Two other shows (non-anime, also without chapters in the files) I have added since last week also have no detected intros.

**EDIT:** Credit detection seems to still work, only intros don't work.

### Jellyfin install method

Docker

### Container image/tag or Jellyfin version

jellyfin/jellyfin:latest

### Operating System

Synology DSM 7.2.2 Update 2 on a DS1520+

### Support Bundle

```shell
* Jellyfin version: 10.10.3
* Plugin version: 1.10.10+1048eaf26d7e
* Queue contents: 31224 episodes, 1850 seasons
* Warnings: `None`
* FFmpeg: `okay`

FFmpeg version:

ffmpeg version 7.0.2-Jellyfin Copyright (c) 2000-2024 the FFmpeg developers
built with gcc 12 (Debian 12.2.0-14)
configuration: --prefix=/usr/lib/jellyfin-ffmpeg --target-os=linux --extra-version=Jellyfin --disable-doc --disable-ffplay --disable-ptx-compression --disable-static --disable-libxcb --disable-sdl2 --disable-xlib --enable-lto=auto --enable-gpl --enable-version3 --enable-shared --enable-gmp --enable-gnutls --enable-chromaprint --enable-opencl --enable-libdrm --enable-libxml2 --enable-libass --enable-libfreetype --enable-libfribidi --enable-libfontconfig --enable-libharfbuzz --enable-libbluray --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libopenmpt --enable-libdav1d --enable-libsvtav1 --enable-libwebp --enable-libvpx --enable-libx264 --enable-libx265 --enable-libzvbi --enable-libzimg --enable-libfdk-aac --arch=amd64 --enable-libshaderc --enable-libplacebo --enable-vulkan --enable-vaapi --enable-amf --enable-libvpl --enable-ffnvcodec --enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvdec --enable-nvenc
libavutil      59.  8.100 / 59.  8.100
libavcodec     61.  3.100 / 61.  3.100
libavformat    61.  1.100 / 61.  1.100
libavdevice    61.  1.100 / 61.  1.100
libavfilter    10.  1.100 / 10.  1.100
libswscale      8.  1.100 /  8.  1.100
libswresample   5.  1.100 /  5.  1.100
libpostproc    58.  1.100 / 58.  1.100
```
```


### Jellyfin logs

_No response_

**Comment by XmnwildX12** [comment]
This is sounding very similar to the issue I have been having with my windows install for almost a month. I have done 3 fresh installs of Jellyfin including a fresh install of the operating system and multiple clears of the detections and eventually run into similar issues as yourself.

What I have noticed in the log is the following, not sure if you have the same or not:
[ERR] [89] MediaBrowser.Providers.TV.EpisodeMetadataService: Error in "Probe Provider"
MediaBrowser.Common.FfmpegException: ffprobe failed - streams and format are both null.
[WRN] [36] IntroSkipper.Services.Entrypoint: Chromaprint returned 0 points

Usually a simple restart of the server or clearing the entire season and running the scheduled task again will populate the intro and credit for the new files. But I have to do this every single time there is a new file. I have automatically analyze new media and update missing segments during scan checked. I have another instance of Jellyfin on a seedbox with basically the same files and no issues, so I have been losing my mind over this one...

**Comment by DrMcCoy** [comment]
I don't have those warnings in my logs.

**Comment by DrMcCoy** [comment]
For what it's worth, this seems to be fixed in 1.10.10.12.

**Comment by AbandonedCart** [comment]
That’s great news (especially since there seemed to be no explanation for why it was happening and here was some trouble reproducing the issue)
